.gitignore

```
@@ -0,0 +1 @@
+backend/my_env/
```

```suggestion
/backend/my_env/
```
I think this is the correct way.

---

I had to do a "refresh" Sir because the .gitignore was not working so the "my_env" directory was not being ignored: 
This was my exact issue: https://stackoverflow.com/questions/25436312/gitignore-not-working 
The 5k file changes are all from the my_env files that were previously added by mistake.